WordPressKit/WordPressComRestApi.swift

```diff
@@ -484,10 +484,21 @@ extension WordPressComRestApi {
 
     /// A custom error processor to handle error responses when status codes are betwen 400 and 500
     func processError(response: DataResponse<Any>, originalError: Error) -> WordPressComRestApiEndpointError? {
```

Comment:
Do we have test coverage on this? It looks the same, but this seems like a good opportunity to add some, if not!

Reply:
Yes, they are covered. I have pre-added a few unit tests to expand the test coverage before these real library code changes (here is an example: #690).

Below is a screenshot of test coverage from Xcode. The only uncovered code is `?? [:]` in the red line.
<img width="662" alt="Screenshot 2024-01-30 at 11 01 43 AM" src="https://github.com/wordpress-mobile/WordPressKit-iOS/assets/1101828/c8b7ac4f-cfd7-428c-b62e-45cf1066b91a">